Sources/Quick/QuickMain.swift

```diff
@@ -3,6 +3,13 @@ import XCTest
 // NOTE: This file is not intended to be included in the Xcode project or CocoaPods.
 //       It is picked up by the Swift Package Manager during its build process.
 
+/// When using Quick with swift-corelibs-xctest, automatic discovery of specs and
+/// configurations is not available. Instead, you should create a standalone
+/// executable and call this function from its main.swift file. This will execute
+/// the specs and then terminate the process with an exit code of 0 if the tests
+/// passed, or 1 if there were any failures.
+///
+/// Quick is known to work with the DEVELOPMENT-SNAPSHOT-2016-02-03-a Swift toolchain
```

Comment:
Interesting!

1. nit-pick: This sentence should end with a period.
2. I'm on the fence when it comes to including this as documentation here; we'll have to update it constantly. It is very useful to end-users, though!